tests/SessionTest.php

```diff
@@ -83,34 +101,10 @@ public function testRestart()
         $this->session->restart();
     }
 
-    public function testVisitWithoutRunningSession()
```

Comment:
well, the test you kept describes the wrong interaction with the driver (it does not describe the usage of ``isStarted``.

and I would keep the test ensuring we can visit without starting, as the test does not need to know that the code is calling ``$this->start()`` internally rather than directly calling the driver.